FIT/FITsim/AliFITv8.cxx

```diff
@@ -2041,12 +2041,12 @@ void AliFITv8::SetCablesA(TGeoVolume *stl) {
   // 12 cables extending beyond the frame
   Float_t pcablesextend[3] = {2, 15, 0.245};
   Float_t pcablesextendsmall[3] = {3, 2, 0.245};
-  Float_t *ppcablesextend[] = {pcablesextend, pcablesextend, pcablesextendsmall,
+  Float_t *ppcablesextend[4] = {pcablesextend, pcablesextend, pcablesextendsmall,
```

Comment:
Maybe you should try with `ppcablesextend[4][3]`?

Reply:
Why not even just `ppcablesextend[12]` with a nice loop to fill the elements?